static/css/index.css

```diff
@@ -676,6 +676,16 @@ header {
   color: #fff;
 }
 
+#emailerror {
+  margin-bottom: 15px;
+  color: red;
+  background-color: #ffffffad;
+  padding: 2px 10px;
+  border-radius: 5px;
+  opacity: 0;
+  transition: all 0.5s;
+}
+
 /* end content styles */
 
 /* start media query styles */
```

Comment:
[OPTIONAL] It is not a good practice to use this commented lines  to diferentiate or segment your code since it becomes bulky and occupies unnecessary space as your project become bigger. :pray:  You can consider breaking the code into smaller components and using expressive and meaningful naming conventions for variables and functions instead which is a better practice and easier to read and understand. :+1:

Reply:
- Thanks @jerryowusu 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊